python/ray/worker.py

```diff
@@ -578,24 +598,31 @@ def print_failed_task(task_status):
       Error Message: \n{}
   """.format(task_status["function_name"], task_status["operationid"], task_status["error_message"]))
 
+def error_applies_to_driver(error_key, worker=global_worker):
+  """Return True if the error is for this driver and false otherwise."""
+  # TODO(rkn): Should probably check that this is only called on a driver.
+  DRIVER_ID_LENGTH = 20
+  ERROR_ID_LENGTH = 20
+  # Check that the error key is formatted as in push_error_to_driver.
+  assert len(error_key) == 5 + 1 + DRIVER_ID_LENGTH + 1 + ERROR_ID_LENGTH, error_key
```

Comment:
above, define a global variable ERROR_KEY = b"Error:" (we have a convention for naming here I think and also if we include ":" or not) and replace 5 + 1 by len(ERROR_KEY)